user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+.. _user-guide-user-management-permissions-basic:
+
+Access and Permissions Basics
+=============================
+
+OroCRM provides a lot of functionality that can enhance your CRM process. These may be used by different members of the 
+team. However, it is obvious that information and tools used by a marketing associate are different from 
+those required by support teams, or by the company management. Flexible structure of the access and permissions settings 
+helps to ensure that all the information is safe and available right where it is required, and all the users gain access 
+to the features and capabilities meeting their needs.
+
+This guide is aimed to explain the basic ideas of the access and permission settings. If you are the system 
+administrator, please address the :ref:`Access and Permissions Management <user-guide-user-management-permissions>`
+guide.
+
+What Define the Permissions
+---------------------------
+
+Access and permission rights of each user depend on the following: 
+
+- Organizations and units, to which the user belongs: so users with the same role, working in different offices or 
+  subdivisions may have different access to different information. The list of organizations and units to which a user 
+  belongs is defined by the system administrator in the *"Access Settings"*. 
```

Comment:
Edit: Organizations and units to which the user belongs: so users with the same role, working in different offices or subdivisions, may have different access to different information. The list of organizations and units to which a user  belongs is defined by the system administrator in the *"Access Settings"*.